snippets/auth/index.py

```diff
@@ -587,6 +587,58 @@ def import_without_password():
         print('Error importing users:', error)
     # [END import_without_password]
 
+def init_action_code_settings():
+    # [START init_action_code_settings]
+    action_code_settings = auth.ActionCodeSettings(
+        url='https://www.example.com/checkout?cartId=1234',
+        handle_code_in_app=True,
+        ios_bundle_id='com.example.ios',
+        android_package_name='com.example.android',
+        android_install_app=True,
+        android_minimum_version='12',
+        dynamic_link_domain='coolapp.page.link',
+    )
+    # [END init_action_code_settings]
+    return action_code_settings
+
+def password_reset_link():
+    action_code_settings = init_action_code_settings()
+    display_name = 'Example User'
+    # [START password_reset_link]
+    email = 'user@example.com'
+    link = auth.generate_password_reset_link(email, action_code_settings)
+    # Construct password reset email template, embed the link and send
```

Comment:
This wording is a bit awkward. Maybe something like this would be clearer?

    # Construct password reset email from a template embedding the link, and send
    # using a custom SMTP server.

Reply:
Done